rust/helm/hyperlane-agent/values.yaml

```diff
@@ -40,7 +40,7 @@ hyperlane:
     # Set the log formatting
     format: json
     # Set the log level
-    level: info
+    level: debug
```

Comment:
Flagging that we probably want this for the e2e tests too in case we didn't make that change